layouts/partials/head_scripts/datadog_monitoring.html

```diff
@@ -1,20 +1,18 @@
+<!-- Datadog Monitoring -->
 <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-rum-us.js"></script>
 <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-logs-us.js"></script>
 <script>
 
-  // Only initializing RUM if on preview or live env
-{{ if (or (eq $.Site.Params.environment "live") (eq $.Site.Params.environment "preview")) }}
-    DD_RUM.init({
-      clientToken: '{{ $.Site.Params.datadog.clientToken }}',
-      applicationId: '{{ $.Site.Params.datadog.applicationId }}',
-    });
-{{ end }}
+  // RUM init
+  DD_RUM.init({
+    clientToken: '{{ $.Site.Params.datadog.clientToken }}',
+    applicationId: '{{ $.Site.Params.datadog.applicationId }}',
+  });
```

Comment:
Now you removed the environment check this will now run during local development too. Will that skew the data?

Reply:
It won't since the script shouldn't be inlined in local build since the default is now to false